### ISSUE TITLE
Deprecation warnings with sprockets 3.7.0

### ISSUE THREAD

**Issue by kapso** [ai/autoprefixer-rails#96]
`6.3.7` is showing this warning

```
DEPRECATION WARNING: You are using the a deprecated processor interface #<Proc:0x007fb4e461acd8@/tmp/build_718e0c8618e06e536e6ea041ec86ae07/vendor/bundle/ruby/2.3.0/gems/autoprefixer-rails-6.3.7/lib/autoprefixer-rails/sprockets.rb:37>.
remote:        Please update your processor interface:
remote:        https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
remote:        (called from install at /tmp/build_718e0c8618e06e536e6ea041ec86ae07/vendor/bundle/ruby/2.3.0/gems/autoprefixer-rails-6.3.7/lib/autoprefixer-rails/sprockets.rb:37)
```

**Comment by ai** [comment]
:D Ignore this warning, we already support new 4.0 API, but use it only with Sprockets 4.0.

https://github.com/ai/autoprefixer-rails/blob/master/lib/autoprefixer-rails/sprockets.rb#L41

I will try to make some workaround in next release.

**Comment by kapso** [comment]
great, thanks.

**Comment by abeMedia** [comment]
You could pass `silence_deprecation: true` to `register_postprocessor` on Sprockets 3.

**Comment by ai** [comment]
@abeMedia awesome! This is why I like open source :). Do you want to send PR to save your name? :)

**Comment by abeMedia** [comment]
Alright :smiley: 
Will submit it in a couple of hours.

**Comment by abeMedia** [comment]
Hi again. It appears `silence_depreciation` is only for `register_engine`. However I Just had a look at the [sprockets code](https://github.com/rails/sprockets/blob/3.x/lib/sprockets/processing.rb#L242) and it's actually even simpler than I thought! All it takes is amending the version number in your code from 4 to 3 (as the new interface is already supported in sprockets 3 - hence the depreciation notice). Will submit a PR now.

**Comment by ai** [comment]
@abeMedia awesome. Thanks. But I will release it only on next autoprefixer.js release (it will be on next week).

**Comment by ai** [comment]
Released in 6.4.
